libs/ui-components/src/components/Fleet/CreateFleet/steps/UpdatePolicyStep.tsx

```diff
@@ -26,124 +21,31 @@ const UpdatePolicyStep = ({ isReadOnly }: { isReadOnly: boolean }) => {
   const { t } = useTranslation();
 
   const {
-    values: { rolloutPolicy, disruptionBudget, updatePolicy },
-    setFieldValue,
+    values: { useBasicUpdateConfig, rolloutPolicy, disruptionBudget, updatePolicy },
   } = useFormikContext<FleetFormValues>();
 
-  const [forceShowAdvancedMode, setForceShowAdvancedMode] = React.useState<boolean>(false);
-  const [alertSwitchToBasic, setAlertSwitchToBasic] = React.useState<StepSetting>();
-  const hasAdvancedMode = rolloutPolicy.isAdvanced || disruptionBudget.isAdvanced || updatePolicy.isAdvanced;
-
-  const onSettingsChange = (setting: StepSetting, toAdvanced: boolean) => {
-    if (toAdvanced) {
-      switch (setting) {
-        case 'all-settings':
-          setForceShowAdvancedMode(true);
-          break;
-        case 'rollout-policies':
-          setFieldValue('rolloutPolicy.isAdvanced', true);
-          void onChangePolicyType(true);
-          break;
-        case 'disruption-budget':
-          setFieldValue('disruptionBudget.isAdvanced', true);
-          void onChangeDisruptionBudget(true);
-          break;
-        case 'update-policies':
-          setFieldValue('updatePolicy.isAdvanced', true);
-          void onChangeUpdatePolicy(true);
-          break;
-      }
-    } else {
-      setAlertSwitchToBasic(setting);
-    }
-  };
-
-  const onChangePolicyType = async (toAdvanced: boolean) => {
-    await setFieldValue('rolloutPolicy', {
-      isAdvanced: toAdvanced,
-      batches: toAdvanced ? [getEmptyInitializedBatch()] : [],
```

Comment:
There's an issue now, where it's possible to get to the Review page with a rollout policy that has no batches. Before, the empty batch would be initialized by default.

When attempting to PATCH the fleet, the backend returns an error 400 because of that.

Reply:
Updated, I am adding the init batch to init values if there isn't any.